src/lib.rs

```diff
@@ -15,6 +15,7 @@
 //! # extern crate icalendar;
 //! # use chrono::*;
 //! # use icalendar::*;
+//! # fn main() {
```

Comment:
Yup, ```cargo test``` fails. What's happening is this: https://doc.rust-lang.org/stable/book/documentation.html#documentation-as-tests

It is putting the extern crate and use statements inside of main, so main has to be explicit